drivers/SmartThings/matter-pump/profiles/pump-level.yml

```diff
@@ -0,0 +1,21 @@
+name: pump-level
+components:
+- id: main
+  capabilities:
+  - id: switch
+    version: 1
+  - id: switchLevel
+    version: 1
+  - id: pumpOperationMode
+    version: 1
+  - id: pumpControlMode
+    version: 1
+  - id: firmwareUpdate
+    version: 1
+  - id: refresh
+    version: 1
+  categories:
+  - name: Irrigation
+metadata:

```

Comment:
@HunsupJung why is pump using a custom presentation?

Reply:
Removed pump from this PR as discussed and we can handle this open item separately